src/js/index.js

```diff
@@ -277,11 +291,38 @@ class RegularTableElement extends RegularViewEventModel {
         };
 
         console.assert(VIRTUAL_MODES.indexOf(virtual_mode) > -1, `Unknown virtual_mode ${virtual_mode};  valid options are "both" (default), "horizontal", "vertical" or "none"`);
+        /** @private */
         this._virtual_mode = virtual_mode;
+        /** @private */
         this._invalid_schema = true;
+        /** @private */
         this._view_cache = {view: dataListener, config, schema};
         this._setup_virtual_scroll();
     }
+
+    /**
+     * This func only exists to provide hints to doc compulation tools.
+     * Should never be run, and even if it is the body of the func will
+     * never execute.
+     *
+     * @internal
+     * @private
+     * @memberof RegularTableElement
+     */
+    __noop_jsdoc_hints() {
```

Comment:
>  Is this just because `draw()` is on super class?

That is exactly it. I was trying to avoid having to make any changes to run-time code that are purely for the benefit of the doc compiler. But if your preferred poison here is a pass-thru method that just calls `super()`, I can go with that instead